user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/TriggerFeed.md

```diff
@@ -4,10 +4,12 @@ uid: DashboardTriggerFeed
 
 # Trigger feed
 
-This dashboard feed allows you to trigger other components either manually or automatically. Available from DataMiner 10.2.0/10.1.1 onwards.
+This feed (available from DataMiner 10.2.0/10.1.1 onwards) allows you to trigger other components either manually or automatically.
 
 At present, this feed can only be used as a filter feed for a component using a query feed. It will function as a refresh trigger for that component.
```

Comment:
I would replace "using a query feed" with "display a query result". 
You could then also reference that later "... as a refresh trigger for that query".